src/ConsoleConnector/Commands/Options/ParamName.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Autodesk.DataExchange.ConsoleApp.Commands.Options
+{
+    internal class ParamName:CommandOption
```

Comment:
Please rename this to ParameterName, so that it is more uniform with ParameterValue. Increases readability of code.

Reply:
Done.